stack-lts-12.4.yaml

```diff
@@ -0,0 +1,27 @@
+resolver: lts-12.4
+
+packages:
+- ./
+- src/pgf-binary/
+- src/runtime/haskell-bind/
+- src/runtime/haskell/
+- src/tools/
+# Failed building:
+# - src/server/
+# - src/example-based/
+
+extra-deps:
+# Non-standard clone of `cgi` needed because of too strict version
```

Comment:
Can we find a better solution to this? Maybe just change the dependency in `gf.cabal`?